<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputTestWdg</class>
 <widget class="QWidget" name="InputTestWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <widget class="QFrame" name="wgHeader">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(220,220,255);
border-radius: 5px;</string>
      </property>
      <widget class="QLabel" name="lbChildSurname">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>94</y>
         <width>100</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Child surname</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leSurname">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="tbPrevChild">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>34</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="fingram.qrc">
         <normaloff>:/resources/toleft.png</normaloff>:/resources/toleft.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="tbNextChild">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>100</y>
         <width>34</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="fingram.qrc">
         <normaloff>:/resources/toright.png</normaloff>:/resources/toright.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="leName">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="leDateBirth">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>110</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbChildName">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>94</y>
         <width>50</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbChildDB">
       <property name="geometry">
        <rect>
         <x>352</x>
         <y>94</y>
         <width>90</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Date of birth</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leChildId">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>59</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>2</y>
         <width>642</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Testing date</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deDate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbGroup">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Group</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leGroup">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbForm">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Questionnaire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leForm">
            <property name="minimumSize">
             <size>
              <width>542</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbNote">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Note</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leNote">
            <property name="minimumSize">
             <size>
              <width>542</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgQuestions" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <widget class="QPushButton" name="pbPrevQuestion">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>75</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="fingram.qrc">
         <normaloff>:/resources/toleft.png</normaloff>:/resources/toleft.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pbNextQuestion">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>60</y>
         <width>75</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="fingram.qrc">
         <normaloff>:/resources/toright.png</normaloff>:/resources/toright.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lbQuestion">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>139</y>
         <width>480</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background : rgba(80,80,0,30);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="leQuestion">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>461</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbQuest">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>122</y>
         <width>63</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#005500;&quot;&gt;Questing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbQuestNum">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>122</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbImage">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>480</width>
         <height>122</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background : rgba(80,80,0,30);
color :  rgba(60,60,0,20);
font-family: sans-serif;
font-size: 20px; 
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>No picture</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgAnswersMain" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbAnswersCaption">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#005500;&quot;&gt;Answer options&lt;/span&gt; (check the box opposite  to the correct one):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="fingram.qrc"/>
 </resources>
 <connections/>
</ui>
